zos.goerli.json

```diff
@@ -1,183 +0,0 @@
-{
```

Comment:
We need versions for this network (and all the others) in the openzeppelin folder